### PR TITLE
More test fixes

### PR DESCRIPTION
**Pull request by StefanBruens**

### CODE REVIEW

**Comment by Metallicow** [comment]
It would be nice if you referenced in the PullREQz description of what RELz wxPy version changes these commits are fixing from(in summary). 
Ex: 4.0.x->4.1.x etc or whatever and relevant infos.

Use a bulleted or numered list please:
* Change #Bla0 blaaaaaaaaa
* Change #Bla1 blaaa
* Change #Bla2 wxPython is AWESOME!

Thanks in advance.

**Comment by StefanBruens** [comment]
> It would be nice if you referenced in the PullREQz description of what RELz wxPy version changes these commits are fixing from(in summary).

Can you say that again in plain english, please?

**Comment by Metallicow** [comment]
`No description provided.` - What readers see.
maybe this is an issue with github....

I don't speak for others, but yes,. Please, I speak American English.
Speak some more.
